### PR TITLE
Cnec get FRM and minor udpate on build loopflow 

### PR DESCRIPTION
**Pull request by pengbo-ruidian**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

- get FRM is available in SimpleCnec but in Cnec interface => add it in Cnec interface
- build loopflow from reference flow map and loopflow shift map => change iteration entry from one map to another

### CODE REVIEW

File: loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java [hunk 1]
```diff
@@ -199,10 +199,10 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
 
     public Map<String, Double> buildLoopFlowsFromReferenceFlowAndLoopflowShifts(Map<Cnec, Double> frefResults, Map<Cnec, Double> loopFlowShifts) {
         Map<String, Double> loopFlows = new HashMap<>();
-        for (Map.Entry<Cnec, Double> entry : frefResults.entrySet()) {
+        for (Map.Entry<Cnec, Double> entry : loopFlowShifts.entrySet()) {
             Cnec cnec = entry.getKey();
-            double loopflowShift = requireNonNull(loopFlowShifts.get(cnec), () -> "No loopflowShifts value for " + cnec.getId());
-            loopFlows.put(cnec.getId(), entry.getValue() - loopflowShift);
+            double fref = requireNonNull(frefResults.get(cnec), () -> "No reference flow value for " + cnec.getId());
+            loopFlows.put(cnec.getId(), fref - entry.getValue());
```
Comment (alexandrem-rte) [inline loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java:205]: why did you change this?

Comment (pengbo-ruidian) [reply loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputation.java:205]: there is non null  check in one but not in the other map
